nss-day-cohort-51/nss-day-cohort-51.github.io
# Mw fix alien coming soon alignment

# Description
Modal: on the Coming Soon image for students without capstone videos, the alien head was off center. I re-drew the image and upload the even-steven version.  

## Type of change


# Testing Instructions
- pull down the branch and serve the website in the browser
- Click the Learn More button on any student who has not have a video embedded
- The alien head on the coming soon image should be centered in its frame. the modified image is a GIF to preserve resolution and keep a super-small file size


ADDITION:
Added a "More Details" button on the Event Card.
Added styling to fall in-line with existing buttons.

# Checklist:
